browser-sign-in/OktaBrowserSignIn.xcodeproj/xcshareddata/xcschemes/OktaBrowserSignIn.xcscheme

```diff
@@ -38,38 +52,13 @@
       </MacroExpansion>
       <EnvironmentVariables>
          <EnvironmentVariable
-            key = "LOGIN"
-            value = "$(LOGIN)"
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "FIRST_NAME"
-            value = "$(FIRST_NAME)"
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "LOGOUT_REDIRECT_URI"
-            value = "$(LOGOUT_REDIRECT_URI)"
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "OKTA_URL"
-            value = "$(OKTA_URL)"
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "CLIENT_ID"
-            value = "$(CLIENT_ID)"
-            isEnabled = "YES">
-         </EnvironmentVariable>
-         <EnvironmentVariable
-            key = "REDIRECT_URI"
-            value = "$(REDIRECT_URI)"
+            key = "E2E_USERNAME"
+            value = "${E2E_USERNAME}"
```

Comment:
If this works, awesome.  Though IIRC these string values need parenthesis, instead of curly braces.  e.g. `value = "$(E2E_USERNAME)"

Reply:
Good callout, its odd because it works but I've updated this.